Tools/scripts/build_binaries_history.py

```diff
@@ -80,6 +83,41 @@ def record_build(self, hash, tag, vehicle, board, frame, bare_path, start_time,
                   "values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)",
                   (hash, tag, vehicle, board, frame, text, data, bss, start_time, duration))
         c.commit()
+        if vehicle == "Rover":
```

Comment:
Shouldn't we generate this *out* of the history, rather than recording it as a separate step?

i.e. instead of passing a json filepath in here, leave recording as-is but create the json file out of the database.  That will allow us to regenerate these if we want to add more information later, that sort of thing.